about.MD

```diff
@@ -2,10 +2,9 @@
 layout: page
 title: About
 lang: en
-permalink: /about/
```

Comment:
it appears that the deployed page works without those permalink, but when running Jekkyl locally I seem to need them. So unless this causes an issue for you, I'd prefer to leave them in place